map/mwm_url.cpp

```diff
@@ -209,7 +209,7 @@ void ParsedMapApi::ParseMapParam(std::string const & key, std::string const & va
       return;
     }
 
-    points.push_back({.m_lat = lat, .m_lon = lon});
+    points.push_back({lat, lon, "" /* m_name */, "" /* m_id */, "" /* m_style */});
```

Comment:
What is wrong here again? :)
Who and what warning prints here?

Reply:
It's a C++20 extension. `-Wpedantic` enables this warning

Reply:
No, emplace_back requires a constructor to be available for the type being emplaced. If the type does not have an appropriate constructor that matches the arguments you pass to emplace_back, the compiler will report an error, as it cannot create the object properly.